lib/shoulda/matchers/active_model/validate_absence_of_matcher.rb

```diff
@@ -113,7 +113,7 @@ def value
             case column_type
             when :integer, :float then 1
             when :decimal then BigDecimal(1, 0)
-            when :datetime, :time, :timestamp then Time.now
+            when :datetime, :time, :timestamp then Time.zone.now
```

Comment:
This makes sense, but again, are there issues you noticed when using the gem?

Reply:
I did not face any issues while using gem.

Reply:
Okay, fair enough. I'm surprised Rubocop didn't catch this, actually. What about `Time.current`? It's shorter :)

Reply:
Agreed with what you just said, since `Time.current` is just a shorter version of `Time.zone.now`, we can use that instead.